recipes/epa-ng/meta.yaml

```diff
@@ -0,0 +1,36 @@
+package:
+  name: epa-ng
+  version: "0.3.3"
+
+build:
+  number: 1
+
+source:
+  url: https://github.com/Pbdas/epa-ng/archive/v0.3.3.tar.gz
+  md5: 881d024877f3cfa632d344373e119f9d
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - libtool
+    - flex
+    - bison
+    - m4
+    - cmake==3.11
+    - automake
+    - autoconf
+  run:
+    - zlib
+
+test:
+  commands:
+    - epa-ng -h > /dev/null
+
+about:
+  home: https://github.com/Pbdas/epa-ng
+  license: GNU Affero General Public License v3.0
```

Comment:
GPL requires that the license be distributed with the packages. Please add `license_file:`

Reply:
Yes. It needs to be distributed with the binaries as well. By specifying `license_file: LICENSE` conda-build knows which file to add to the package (ends up in the `info` folder).

Reply:
Ok thanks - I added it